mithril.js

```diff
@@ -1318,7 +1372,7 @@
 
 	function checkPrevented(component, root, index, isPrevented) {
 		if (!isPrevented) {
-			m.redraw.strategy("all")
+			m.redraw.strategy("diff")
```

Comment:
I think this is what's breaking tests. Would it be possible for this to work with `m.redraw.strategy() === "all'`?